from graph_exception import Node1

class Test:
    def test_setup(self):
        root = Node1()